core/src/main/java/org/apache/carbondata/core/util/DeleteLoadFolders.java

```diff
@@ -184,11 +184,7 @@ private static boolean checkIfLoadCanBeDeleted(LoadMetadataDetails oneLoad,
 
   private static boolean checkIfLoadCanBeDeletedPhysically(LoadMetadataDetails oneLoad,
```

Comment:
this function is calling another function directly, you can remove this function